### PR TITLE
Update import-api tests to use Authorization header

### PR DESCRIPTION
**Pull request by CarlHembrough**
### What

The import API has been updated to use the identity authentication endpoint provided by Zebedee. The tests have been updated to add the required headers.

Requires import api branch: https://github.com/ONSdigital/dp-import-api/pull/41
Requires identity api stub: https://github.com/ONSdigital/dp-auth-api-stub/pull/2

### How to review

review changes / test

### Who can review

Anyone

### CODE REVIEW

File: importAPI/get_job_test.go [hunk 1]
```diff
@@ -34,16 +34,46 @@ func TestSuccessfullyGetAnImportJob(t *testing.T) {
...
-				importAPI.GET("/jobs/{id}", jobID).Expect().Status(http.StatusNotFound)
+	if err := mongo.Setup(importCreateJobDoc); err != nil {
+		log.ErrorC("Failed to set up test data", err, nil)
+		os.Exit(1)
+	}
 
+	importAPI := httpexpect.New(t, cfg.ImportAPIURL)
+
+	Convey("Given an import job exists", t, func() {
+		Convey("When a request to get the job with id does not exist", func() {
```
Comment (gedge) [inline importAPI/get_job_test.go]: I find this English hard to parse, consider rephrase "... with an id that does not exist" or "... where the id does not exist"?

same for subsequent tests, I'm afraid - perhaps "get the job by ID and..." is appropriate for most, here

File: importAPI/get_job_test.go [hunk 2]
```diff
@@ -34,16 +34,46 @@ func TestSuccessfullyGetAnImportJob(t *testing.T) {
...
+			Convey("Then the response returns status not found (404)", func() {
+				importAPI.GET("/jobs/{id}", uuid.NewV4().String()).
```
Comment (gedge) [inline importAPI/get_job_test.go]: isn't `uuid/NewV4()` being deprecated in favour of `invalidJobId`?

File: importAPI/put_job_test.go [hunk 3]
```diff
@@ -84,8 +133,10 @@ func TestFailureToUpdateAnImportJob(t *testing.T) {
 	Convey("Given an import job exists", t, func() {
 		Convey("When a request to change job state with job id that does not exist", func() {
 			Convey("Then the response returns status not found (404)", func() {
-				importAPI.PUT("/jobs/{id}", uuid.NewV4().String()).WithBytes([]byte(validPUTJobJSON)).
-					WithHeader(tokenName, tokenSecret).Expect().Status(http.StatusNotFound)
+				importAPI.PUT("/jobs/{id}", uuid.NewV4().String()).
```
Comment (gedge) [inline importAPI/put_job_test.go]: prefer `invalidJobID`?

**Review by gedge — CHANGES_REQUESTED** [review]

File: importAPI/get_job_test.go (hunk 1 above)
Comment (CarlHembrough) [reply importAPI/get_job_test.go]: I have revamped all of the import API tests to hopefully be more clear and concise.

File: importAPI/get_job_test.go (hunk 2 above)
Comment (CarlHembrough) [reply importAPI/get_job_test.go]: Good spot, I have changed to use `invalidJobID`

File: importAPI/put_job_test.go (hunk 3 above)
Comment (CarlHembrough) [reply importAPI/put_job_test.go]: Done 👍

**Review by gedge — APPROVED** [review]
